### PR TITLE
Example dummy and fixes for issues when partial cpus were used.

### PR DESCRIPTION
**Pull request by savpek**
- Fixed issue when container has for example 0.5 cpu.
- Add working example with puppeteer.
- Support for resource group tags.

### CODE REVIEW

File: example/puppeteer/test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        await page.type('#userName', 'foo');
+        await page.type('#password', 'bar');
+        await page.waitForSelector('#login-button');
+        await page.click('#login-button');
+    })
+})
+
+after(async () => {
+    await browser.close()
+})
```
Comment (tarlepp) [inline example/puppeteer/test.js]: missing new line on end of the file.

File: example/puppeteer/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npx foo.spec.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "puppeteer": "^1.19.0",
+    "mocha": "6.2.0"
```
Comment (tarlepp) [inline example/puppeteer/package.json:13]: with mocha you're using exact version and with puppeteer not - is there some reason for this?

File: example/puppeteer/.dockerignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (tarlepp) [inline example/puppeteer/.dockerignore:1]: missing new line on the end of the file.

**Review by aeosys — APPROVED** [review]
Seems legit 🎩

**Review by Hekku2 — APPROVED** [review]
Seems fine. Did not test.

**Review by samikaur — APPROVED** [review]

**Pull request merged**